src/yellow/components/button/_index.scss

```diff
@@ -2,126 +2,158 @@
 
 .btn {
   border: 0;
+  border-radius: $size-border-radius-pill;
   cursor: pointer;
-  display: inline-block;
   font-weight: $font-style-weight-bold;
-  letter-spacing: .3px;
-  line-height: 1;
+  min-width: 280px;
   text-align: center;
   text-transform: uppercase;
-  transition: border-color .2s ease-in, box-shadow .2s ease-in, background-color .2s ease-in, color .3s ease;
+  transition: all .2s linear;
 
-  &__primary {
-    border-radius: $size-border-radius-pill;
-    transition: all .2s linear;
+  &--small {
+    font-size: $size-font-xxs;
+    padding: $size-spacing-xxs $size-spacing-m;
+  }
```

Comment:
I was in doubt to create this modifier or already put as the default property of the component.

What do you think?

Reply:
I vote to create the modifier, once that it is a specific version of the button. It make sense.

Reply:
It is my real point that if you try to use the classes on any button without any of the `btn - small` or` btn - large` classes it will probably inherit the font size from the parent element.

Reply:
I think this could be the default properties of the component. The component should has font and padding size previously specified.

Reply:
But I think it's a bit strange, because we need to have a lower size to the default button as well. Maybe a new button with the same `font-size`, but with a different `padding`, `$size-spacing-xxxs $size-spacing-s` for example, what do you think?

Reply:
> But I think it's a bit strange, because we need to have a lower size than the default button as well. Maybe a new button with the same `font-size`, but with a different `padding`, `$size-spacing-xxxs $size-spacing-s` for example, what do you think?

![image](https://user-images.githubusercontent.com/13206817/70649019-b2a75d00-1c22-11ea-962f-e47e53259724.png)

Reply:
I talked to Mari and she talked to the rest of the designers. According to them as a solution per hour we will apply the sizes from small to deefault as well.